build/resources/main/data/npcs.json

```diff
@@ -0,0 +1,25 @@
+[{
+  "type": "NPC",
+  "name": "chad",
+  "description": "The chad tells you to stay away from his princess.",
+  "inventory": [],
+  "active": true,
+  
+  "stat": {
+    "name": "health",
+    "value": 10
+  }
+},
+{
+  "type": "NPC",
+  "name": "princess",
+  "description": "The princess tells you that you were warned to stay away from her!",
```

Comment:
I like the dry, quirky comedy vibe. Try to do that everywhere, but not too generic of course.